rpm/prebuild.sh

```diff
@@ -0,0 +1,37 @@
+set_python3() {
+    python3_default=/usr/bin/python3
+    for v in `seq 9 -1 4`
+    do
+      python3=/usr/bin/python3.$v
+      if [[ -e $python ]] && [[ ! -e $python3_default ]]; then
+        echo "Create symlink for Python 3: $python3_default --> $python3"
+        sudo ln -s $python3_default $python3
```

Comment:
I verified CentOS (6, 7), Fedora (28, 34), openSUSE Leap (15.0, 15.3) and all have `python3` executable in default PATH. So I don't see a reason to do so.

Reply:
script `rpm/prebuild.sh` has been removed